lib/api/valuation.js

```diff
@@ -4,43 +4,84 @@ function getJuneDefensiveSite() {
   return fetch('https://www.fundsquare.net/security/summary?idInstr=256575')
     .then(res => res.text());
 }
+function getJuneDefensivePriceSite() {
```

Comment:
I would like to move these new `invested` method to be moved into a new file `lib/api/invested.js`